src/main/java/com/syntaxphoenix/spigot/smoothtimber/SmoothTimber.java

```diff
@@ -79,7 +90,9 @@ public static void triggerFallEvent(final AsyncPlayerTreeFallEvent event) {
 
     public static void triggerChoppedEvent(final Player player, final Location location, final VersionChanger change, final ItemStack tool,
         final ArrayList<Location> woodBlocks, final int limit) {
-        Bukkit.getPluginManager().callEvent(new AsyncPlayerChoppedTreeEvent(player, location, change, tool, woodBlocks, limit));
+        scheduler.runTaskAsynchronously(() -> {
```

Comment:
Do we need this here? Most likely not

Reply:
for these two async schedules that just call events i don’t remember y i did it tbh it probably throws a warning if it isn’t scheduled like this, i’m on mobile rn bc i’m moving into my dorm but when i get everything setup i can check what happens if it’s removed and i’ll probably remember y i did it

Reply:
As far as I know the 'triggerXXXEvent' methods are called asynchronously already so there is no need for an async scheduler I think, but I can understand that